test

```diff
@@ -0,0 +1,3 @@
+# no longer using update -t 2 because of https://github.com/IATI/iati-datastore/issues/194
```

Comment:
Is this file still used for anything, or can it be removed?

Reply:
no idea. should not be there.